### DISCUSSION TITLE
inspect disable exts dont work

### DISCUSSION THREAD

**Discussion by GlizxyBoi** [3kh0/ext-remover#674] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is there a possible way to fix this?

**Comment by Aka-but-nice** [comment, marked as answer]
> is there a possible way to fix this?

There is one way. I suggest you buy a personal Chromebook. Amazon has really high-quality Chromebooks for almost less than 70 dollars, it's quite an interesting investment
